examples/complete/main.tf

```diff
@@ -270,6 +271,9 @@ module "dedicated_host" {
 #############################################################################
 
 module "slz_vsi_dh" {
+  count                           = var.enable_dedicated_host ? 1 : 0
+  depends_on                      = [module.dedicated_host]
```

Comment:
Not sure why there is an explicit depends_on here? There is an implicit dependency on the next line

Reply:
Oh yes there is no need for explicit depends_on, it is already covered. 
Thank you for the correction. I have removed it.